shared/contract-utils/test/helpers/assertThrow.js

```diff
@@ -0,0 +1,34 @@
+const THROW_ERROR_PREFIX = 'VM Exception while processing transaction: revert'
```

Comment:
Are these changes to `assertThrow()` needed?

As far as I can see, the only change is to remove [this line](https://github.com/aragon/aragon-apps/blob/master/shared/test-helpers/assertThrow.js#L25), which by itself shouldn't cause any problems (unless `error.message` is now non-existent?).

Reply:
Yes, it's needed: the structure of `error.message` is now different, so it never matches the string. Besides, having now `error.reason` with what we are looking for directly is way more elegant and trustworthy.

Reply:
Sounds like this is something we should handle generically across truffle v4 and v5 in contract-helpers then.